### PR TITLE
Travis CI: test against Crystal nightlies

### PR DESCRIPTION
**Pull request by Sija**
SSIA

### CODE REVIEW

**Comment by Sija** [comment]
@veelenga See https://travis-ci.org/github/crystal-ameba/ameba/jobs/683338138 🔴

**Comment by veelenga** [comment]
Yeah, i know. We have [crystal-head](https://github.com/crystal-ameba/ameba/tree/crystal-head) branch with periodic build setup which notifies. I just need to reserve time and look in.

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+
 script:
   - make test
   - crystal tool format --check
   - sed -i -e 's:<.*>::g' README.md
   - crystal docs
+
 deploy:
   provider: pages
-  github_token: $GITHUB_TOKEN
```
Comment (veelenga) [inline .travis.yml:9]: please rever the unrelated changes. Moreover, `github_token` is an alias and imo just works here.

Comment (Sija) [reply .travis.yml:9]: Will do. Being an alias, adds to the Travis CI validation notes, so I reckoned it's better to use the actual key name instead.

**Comment by veelenga** [comment]
Thank you.
